sentry_sdk/integrations/logging.py

```diff
@@ -43,75 +62,76 @@ def sentry_patched_callhandlers(self, record):
             if record.name not in _IGNORED_LOGGERS:
                 integration = Hub.current.get_integration(LoggingIntegration)
                 if integration is not None:
-                    integration._handler.handle(record)
+                    integration._handle_record(record)
             return old_callhandlers(self, record)
 
         logging.Logger.callHandlers = sentry_patched_callhandlers
 
 
-class SentryHandler(logging.Handler, object):
-    def __init__(self, level=DEFAULT_LEVEL, event_level=DEFAULT_EVENT_LEVEL, **kwargs):
-        logging.Handler.__init__(self, level, **kwargs)
-        if event_level is None:
-            self._event_level = None
-        else:
-            self._event_level = logging._checkLevel(event_level)
+def _can_record(record):
+    return record.name not in _IGNORED_LOGGERS
 
+
+def _breadcrumb_from_record(record):
+    return {
+        "ty": "log",
+        "level": _logging_to_event_level(record.levelname),
+        "category": record.name,
+        "message": record.message,
+        "timestamp": datetime.datetime.fromtimestamp(record.created),
+    }
+
+
+def _logging_to_event_level(levelname):
+    return {"critical": "fatal"}.get(levelname.lower(), levelname.lower())
+
+
+class SentryHandler(logging.Handler, object):
```

Comment:
If we stick with two handlers let's call this one `EventHandler` and make a `SentryHandler` alias for existing uses.